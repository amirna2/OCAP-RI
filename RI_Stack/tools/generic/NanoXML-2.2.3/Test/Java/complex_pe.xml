<?xml version='1.0'?>
<!DOCTYPE test [
	<!ELEMENT test (#PCDATA)>
	<!ENTITY % xx '&#37;zz;'>
	<!ENTITY % zz '&#60;!ENTITY tricky "error-prone" >' >
	%xx;
	<!ENTITY % Boolean '(True|False)'>
	<!ATTLIST test foo %Boolean; 'True'>
]>

<test>This sample shows a &tricky; method.</test>

