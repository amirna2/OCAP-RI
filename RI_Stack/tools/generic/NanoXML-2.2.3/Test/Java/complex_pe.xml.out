<test foo="True">This sample shows a error-prone method.</test>